payments-core/src/test/java/com/stripe/android/paymentsheet/PaymentSheetAddPaymentMethodFragmentTest.kt

```diff
@@ -58,6 +60,19 @@ class PaymentSheetAddPaymentMethodFragmentTest {
         }
     }
 
+    @Ignore("Disabled until more payment methods are supported")
```

Comment:
Is this ignore reason correct, or is this disabled until processing is supported?

Reply:
Yes, it's ignored because we're filtering out all non-card payment methods in `BaseSheetViewModel`, so it doesn't pass.

Reply:
The test is testing that we actually set the value, once it's supported we can check the UI state too.

Reply:
Would this work for card and just not sofort?

Reply:
Sorry, this is testing that the RecyclerView is disabled. With a single payment method the RecyclerView is not shown.